### PR TITLE
Change design using material ui

### PR DESCRIPTION
**Pull request by luisbonachera**
Check if the changes made are correct. I have made style changes to the menu buttons.

### CODE REVIEW

File: src/Menu.tsx [hunk 1]
```diff
@@ -66,44 +86,57 @@ const TemporaryDrawer = (props: Props) => {
...
               <ListItemIcon>
-                <VisibilityIcon />
+                <VisibilityIcon className={classes.iconColor} />
               </ListItemIcon>
-              <Link to="/private">Private</Link>
+              <Link to="/private" className={classes.linkColor}>
+                <Typography variant="h6">
+                  Private
+            </Typography>
+              </Link>
```
Comment (dailymp) [inline src/Menu.tsx]: You should fix the identation

File: src/Menu.tsx [hunk 2]
```diff
@@ -112,12 +145,16 @@ const TemporaryDrawer = (props: Props) => {
...
               </ListItemIcon>
-              <Link to="/courses">Courses</Link>
+              <Link to="/courses" className={classes.linkColor}>
+                <Typography variant="h6">
+                  Courses
+            </Typography>
+              </Link>
             </ListItem>
           </List>
         </>
```
Comment (dailymp) [inline src/Menu.tsx]: I can see you just modified menu styles, but not the rest of the components, can you modify every component, for example: Public, Private, Courses, etc.?

File: src/Menu.tsx [hunk 3]
```diff
@@ -15,6 +15,12 @@ import PublicIcon from '@material-ui/icons/Public';
 import VisibilityIcon from '@material-ui/icons/Visibility';
 import { Link } from 'react-router-dom';
 import Auth from './Auth/Auth';
+import Typography from "@material-ui/core/Typography";
+import { NONAME } from 'dns';
+
```
Comment (dailymp) [inline src/Menu.tsx]: Are you really using NONAME?

File: src/Menu.tsx [hunk 4]
```diff
@@ -24,14 +30,31 @@ const useStyles = makeStyles(theme => ({
...
+    }
+  },
+  // hover: {
+  //   "&:hover": {
+  //     backgroundColor: '#3f51b5',
+  //     linkStyle: {
+  //       color: '#757de8',
+  //     },
+  //   },
+  // }
```
Comment (dailymp) [inline src/Menu.tsx]: this commented code shouldn't be here

**Review by dailymp — CHANGES_REQUESTED** [review]

**Comment by dailymp** [comment]
You must resolve conflicts first, read this: https://help.github.com/en/articles/resolving-a-merge-conflict-on-github

File: src/Menu.tsx (hunk 3 above)
Comment (luisbonachera) [reply src/Menu.tsx]: I am not using NONAME

File: src/Menu.tsx (hunk 1 above)
Comment (luisbonachera) [reply src/Menu.tsx]: Everything is already indented

File: src/Menu.tsx (hunk 2 above)
Comment (luisbonachera) [reply src/Menu.tsx]: I have made some slight changes of styles using material-ui in the Home, Private and Public components.

**Pull request merged**